android/src/main/java/com/thebluealliance/androidclient/gcm/notifications/BaseNotification.java

```diff
@@ -25,12 +26,16 @@
 import com.thebluealliance.androidclient.listitems.ListElement;
 import com.thebluealliance.androidclient.models.StoredNotification;
 import com.thebluealliance.androidclient.receivers.NotificationChangedReceiver;
+import com.thebluealliance.androidclient.viewmodels.GenericNotificationViewModel;
+import com.thebluealliance.androidclient.viewmodels.ViewModelRenderer;
 
 import java.text.DateFormat;
 import java.util.Calendar;
 import java.util.Date;
 
-public abstract class BaseNotification extends ListElement {
+import io.nlopez.smartadapters.views.BindableLayout;
+
+public abstract class BaseNotification<VIEWMODEL> extends ListElement implements ViewModelRenderer<VIEWMODEL, Void> {
```

Comment:
Since you're always passing `Void` as the args type and if you do what I suggested below by enforcing only a superclass type for ViewModel, you won't need the generic bit here. 

Just add `public abstract ParentViewModel getViewModel(Context c)` to this class